WORKSPACE

```diff
@@ -90,16 +63,25 @@ node_repositories(
 
 yarn_install(
     name = "npm",
-    data = ["//:postinstall.tsconfig.json"],
+    data = [
+        "//:postinstall.tsconfig.json",
+        # Need a reference to @angular here so that Bazel sets up the
+        # external repository before calling yarn_install
+        "@angular//:LICENSE",
```

Comment:
This shouldn't be necessary in this repo. Its in the angular repo since the integration/bazel package.json references the generated @angular/bazel npm package.

Reply:
it was necessary here, broke otherwise because of the postinstall of @angular/bazel looking for the @angular repo

Reply:
right. it's still running check_version.js. that's an unfortunate leakage